Kosystem/wwwroot/css/blazor.css

```diff
@@ -0,0 +1,19 @@
+﻿
+#blazor-error-ui {
+    background: lightyellow;
+    bottom: 0;
+    box-shadow: 0 -1px 2px rgba(0, 0, 0, 0.2);
+    display: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=5636989)